docs/RFCS/00000000_template.md

```diff
@@ -30,7 +30,8 @@ Explain the proposal as if it was already included in the project and
 you were teaching it to another CockroachDB programmer. That generally means:
 
 - Introducing new named concepts.
-- Explaining the feature largely in terms of examples.
+- Explaining the feature largely in terms of examples. If applicable, stick to PM provided user stories
```

Comment:
I'm wary about saying "stick to PM provided user stories" since right now we haven't had the bandwidth to fully flesh out features. maybe "if applicable, tie out (or maybe align) the RFC with PM provided user stories"?

Reply:
:+1: on nate's suggestion. However, let's not completely forget what's driving what in an open source project. May I suggest instead:

"""

- Explaining the feature largely in terms of examples. Take into account that a product manager (PM) will want to connect back the work introduced by the RFC with user stories. Whenever practical, do ask PMs if they already have user stories that relate to the proposed work, and do approach PMs to attract user buy-in and mindshare if applicable.

"""